### PR TITLE
Add czech holidays

### PR DESCRIPTION
**Pull request by dfridrich**

### CODE REVIEW

File: src/Yasumi/Provider/CzechRepublic.php [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+use DateTime;
+use DateTimeZone;
+use Yasumi\Holiday;
+
+/**
+ * Provider for all holidays in Czech republic.
+ *
+ * Class CzechRepublic
+ * @package Yasumi\Provider
+ * @author Dennis Fridrich <fridrich.dennis@gmail.com>
```
Comment (stelgenhof) [inline src/Yasumi/Provider/CzechRepublic.php]: I believe you have copied the Germany Test classes :) No problem,but maybe good to remove/adjust all references to the other class and authors.

**Comment by stelgenhof** [comment]
Thanks for the Czech Republic Holiday Provider! Let me have a look at your PR; I'd like to have it then part of the 1.4.0 version.

File: src/Yasumi/Provider/CzechRepublic.php (hunk 1 above)
Comment (dfridrich) [inline src/Yasumi/Provider/CzechRepublic.php]: @stelgenhof you are right, I recycled Germany version :) Files are repaired now.

**Comment by stelgenhof** [comment]
Hi @dfridrich 
The PR is looking good, but there are some conflicts. Likely because there were some changes to the master branch after your PR. Could you check these please? Then afterwards I will merge your PR!

**Comment by dfridrich** [comment]
@stelgenhof Conflicts are resolved

**Pull request merged**